autoload/ctrlsf.vim

```diff
@@ -35,6 +36,14 @@ func! s:ExecSearch(args) abort
     endif
 
     call ctrlsf#db#ParseAckprgResult(output)
+
+    " Only populate and open the quickfix window
+    if s:only_quickfix
```

Comment:
`s:ExecSearch` should accept `only_quickfix` as an argument and use it, rather than script-scoped `only_quickfix`. (for `ctrlsf#Update()`'s usage)